1603717726455_agency_seed.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE "public"."User"("email" text NOT NULL, "agencyId" uuid NOT NULL, PRIMARY KEY ("email") );
```

Comment:
"User" is very vague. Are all users of this app equal? Are there not multiple personas with different data attached?

Reply:
Right now we haven't detected anything different with the personas in terms of data